pages/product-listing/products.scss

```diff
@@ -0,0 +1,251 @@
+/* =========== Products Page =========== */
+body {
+    background-color: #e5e7eb;
+}
+
+.header {
+    background-color: var(--body-color);
+}
+
+.nav__logo {
+    color: var(--text-color);
+}
+
+.navbar {
+    .nav_btns {
+        a {
+            font-size: var(--h1-font-size);
+            color: var(--dark-color);
+        }
+    }
+}
+
+.main {
+    padding: 0 auto;
+    margin-top: calc(var(--header-height) + var(--sp-3));
+
+    .sidebar {
+        display: none;
+
+        text-transform: uppercase;
+        background-color: var(--body-color);
+        width: 20%;
+        max-width: 18rem;
+        position: fixed;
+        overflow: auto;
+        border-radius: 0.25rem;
+
+        box-shadow: 0px 0px 5px 3px rgba(0, 0, 0, 0.08);
+        -webkit-box-shadow: 0px 0px 5px 3px rgba(0, 0, 0, 0.08);
+        -moz-box-shadow: 0px 0px 5px 3px rgba(0, 0, 0, 0.08);
```

Comment:
You can declare this as global as well..

Reply:
Thanks, I did the changes in my last commit.